aten/src/ATen/native/sparse/cuda/SoftMax.cu

```diff
@@ -575,10 +566,10 @@ void cuda_sparse_coo_softmax_backward(
   if (nvalues > 0 && pool_size > 0) {
     cuda_sparse_coo_softmax_backward_kernel<scalar_t, LogSoftMax>
         <<<grid_size, block_size, 0, stream>>>(
-            sorted_indices.data_ptr<int64_t>(),
+            sorted_indices.template data_ptr<int64_t>(),
             pool_size,
-            pool_sizes.data_ptr<int64_t>(),
-            pool_offsets.data_ptr<int64_t>(),
+            pool_sizes.template data_ptr<int64_t>(),
+            pool_offsets.template data_ptr<int64_t>(),
```

Comment:
I hope this doesn't increase code size much.

Reply:
Why?